CHANGELOG.md

```diff
@@ -41,9 +41,12 @@ Our versioning strategy is as follows:
   * Promo component ([#1897](https://github.com/Sitecore/jss/pull/1897))
   * PageContent component ([#1905](https://github.com/Sitecore/jss/pull/1905))
   * Navigation component ([#1894](https://github.com/Sitecore/jss/pull/1894))
+* `[sitecore-jss-angular]` Placeholder component will now render metadata code blocks when in Metadata Edit mode ([#1918](https://github.com/Sitecore/jss/pull/1918))
 
 ### 🛠 Breaking Change
 
+* `[templates/angular]` JssContextService now relies on JssStateService for state management. ([#1918](https://github.com/Sitecore/jss/pull/1918))
```

Comment:
New _JssStateService_ should be mentioned in a changelog as a feature (since it's a new service) and explanation of the usage

Reply:
But it is a breaking change that requires customers to adjust also?

Reply:
It's a new service that we introduced, so we need to provide more information on that, where it's available and why it was introduced. Customer can think that this change is not required to be done by them, currently there is no mentioning that this service is referenced by the components that are introduced on SDK level
I agree that it's a breaking change, I forgot that we started to reference this service in our SDK as well.